domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/datamapper/domene/SvpMapper.java

```diff
@@ -0,0 +1,145 @@
+package no.nav.foreldrepenger.melding.datamapper.domene;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import no.nav.foreldrepenger.melding.beregning.BeregningsresultatAndel;
+import no.nav.foreldrepenger.melding.beregning.BeregningsresultatFP;
+import no.nav.foreldrepenger.melding.beregning.BeregningsresultatPeriode;
+import no.nav.foreldrepenger.melding.beregningsgrunnlag.AktivitetStatus;
+import no.nav.foreldrepenger.melding.datamapper.brev.FritekstmalBrevMapper.Brevdata;
+import no.nav.foreldrepenger.melding.datamapper.domene.sammenslåperioder.PeriodeBeregner;
+import no.nav.foreldrepenger.melding.datamapper.domene.sammenslåperioder.PeriodeMergerSvp;
+import no.nav.foreldrepenger.melding.dokumentdata.DokumentFelles;
+import no.nav.foreldrepenger.melding.integrasjon.dokument.felles.FellesType;
+import no.nav.foreldrepenger.melding.typer.DatoIntervall;
+import no.nav.foreldrepenger.melding.uttak.svp.PeriodeIkkeOppfyltÅrsak;
+import no.nav.foreldrepenger.melding.uttak.svp.SvpUttakResultatPeriode;
+import no.nav.foreldrepenger.melding.uttak.svp.SvpUttakResultatPerioder;
+import no.nav.foreldrepenger.melding.uttak.svp.SvpUttaksresultat;
+import no.nav.foreldrepenger.melding.virksomhet.Arbeidsgiver;
+
+public class SvpMapper {
+
+    private SvpMapper() {
+    }
+
+    public static int finnAntallRefusjonerTilArbeidsgivere(BeregningsresultatFP beregningsresultatFP) {
+        return (int) beregningsresultatFP.getBeregningsresultatPerioder().stream()
+                .map(BeregningsresultatPeriode::getBeregningsresultatAndelList)
+                .flatMap(List::stream)
+                .filter(BeregningsresultatAndel::erArbeidsgiverMottaker)
+                .map(beregningsresultatAndel -> beregningsresultatAndel.getArbeidsgiver().map(Arbeidsgiver::getIdentifikator).orElse(beregningsresultatAndel.getArbeidsforholdRef().getReferanse()))
+                .distinct().count();
+    }
+
+    public static SvpUttaksresultat utvidOgTilpassBrev(SvpUttaksresultat svpUttaksresultat, BeregningsresultatFP beregningsresultatFP
+                                                       //List<BeregningsgrunnlagPeriode> beregningingsgrunnlagperioder,
```

Comment:
slette dett parameter hvis det er ikke behov.

Reply:
Det blir kanskje behov for den senere. Bare merger inn nå så de kan starte å teste